controller/array_action/array_mediator_ds8k.py

```diff
@@ -525,10 +524,11 @@ def _create_flashcopy(self, source_volume_id, target_volume_id, options=None):
                                                                          target_volume_id))
             else:
                 raise ex
-        if not self.validate_flashcopy(api_flashcopy.id):
+        flashcopy_state = self.get_flashcopy_state(api_flashcopy.id)
+        if not flashcopy_state == FLASHCOPY_STATE_VALID:
             self._delete_flashcopy(api_flashcopy.id)
-            logger.info("Flashcopy is not in a valid state")
-            raise ValueError
+            raise ValueError("Flashcopy state is not correct. expected: {} , got: '{}'.".format(FLASHCOPY_STATE_VALID,
```

Comment:
align all `{}` to either be or not be surrounded with apostrophes

Reply:
done